pylossless/pipeline.py

```diff
@@ -765,9 +823,21 @@ def _flag_volt_std(self, flag_dim, threshold=5e-5, picks="eeg"):
         on. You may need to assess a more appropriate value for your own data.
         """
         epochs = self.get_epochs(picks=picks)
-        above_threshold = _threshold_volt_std(
-            epochs, flag_dim=flag_dim, threshold=threshold
-        )
+        # So yes this add a few LOC, but IMO it's worth it for readability
```

Comment:
I suggest avoiding using such acronyms in code documentation. I have no idea what LOC and IMO mean, so I have to Google them and then guess what is meant among the multiple LOC acronym definitions returned by good. Creates confusion, which is the opposite of the objective of code comments.

Reply:
Ok I will try to be better about that : ) 

Fixed in [8e5eb40](https://github.com/lina-usc/pylossless/pull/214/commits/8e5eb40a93821f625dbc0e73b652794560580c36)